akanda/router/drivers/iptables.py

```diff
@@ -325,6 +326,16 @@ def _build_floating_ips(self, config):
                         ), ip_version=4
                     ))
 
+        if rules:
```

Comment:
So if *at least* one floating IP exists, we set up a single rule for the entire v4 cidr